### PR TITLE
Change: use vcpkg integration in Visual Studio

### PR DESCRIPTION
**Pull request by glx22**
Removes the need for us to provide OpenTTD-useful package

### CODE REVIEW

**Review by TrueBrain — CHANGES_REQUESTED** [review]
Sweet :D

Please also update https://github.com/OpenTTD/OpenTTD/blob/master/README.md#70-compiling with the new instructions in the Pull Request. Otherwise a lot of people will be a bit confused :)

File: src/stdafx.h [hunk 1]
```diff
@@ -245,12 +245,17 @@
 		#define fseek _fseeki64
 	#endif /* _WIN64 || WIN64 */
 
-	/* This is needed to zlib uses the stdcall calling convention on visual studio */
-	#if defined(WITH_ZLIB) || defined(WITH_PNG)
-		#if !defined(ZLIB_WINAPI)
-			#define ZLIB_WINAPI
-		#endif
-	#endif
+#	if defined(WITH_ZLIB)
```
Comment (TrueBrain) [inline src/stdafx.h:249]: Possibly a comment to indicate why this is needed?

**Comment by glx22** [comment]
https://github.com/OpenTTD/OpenTTD/blob/master/docs/Readme_Windows_MSVC.txt will need an update too, but it's a lot bigger than the scope of this PR (and I'm not good at documentation)

**Review by TrueBrain — CHANGES_REQUESTED** [review]
Sorry, I disagree; it is not bigger than the scope of this PR to make that file at least factual correct (or at least for the parts references in this PR). Even if you are not good at documentation, doing what you did for README.md is always the least we can do :) If we don't fix things like that, we make it a lot harder for new people, which is counterproductive :)

**Comment by TrueBrain** [comment]
Added the documentation (and this PR) as part of #6991. Let's continue there :)

**Pull request closed without merging**